static/cfg-view.js

```diff
@@ -156,6 +157,16 @@ define(function (require) {
             this.saveState();
         }
     };
+    
+    Cfg.prototype.closeCfg = function () {
+         _.defer(function (self) {
+                self.container.close();
+            }, this);
+    };
+    
+    Cfg.prototype.onBinaryMode = function() {
```

Comment:
Why not just call closeCfg in the binding instead of having this indirection?

Reply:
following naming convention onEventName but this naming doesn't tell what's the function is doing. so I added one level.